xtext/org.icyphy.linguafranca/src/org/icyphy/generator/GeneratorBase.xtend

```diff
@@ -1022,34 +1034,31 @@ abstract class GeneratorBase extends AbstractLinguaFrancaValidator {
      * @return A ProcessBuilder object if the command was found or null otherwise.
      */
     protected def createCommand(String cmd) {
-        return createCommand(cmd, #[])
+        return createCommand(cmd, #[], findCommandEnv(cmd))
     }
     
     /**
-     * Creates a ProcessBuilder for a given command and its arguments.
+     * Tries to find the command.
```

Comment:
Better documentation for this would be:

Return ExecutionEnvironment.NATIVE if the specified command is directly executable on the current host
and ExecutionEnvironment.BASH if the command must be executed within a bash shell.
The latter occurs, for example, if the specified command is not in the native path but is in the path
specified by the user's bash configuration file.
If the specified command is not found in either the native environment nor the bash environment,
then this reports and error and returns null.